pkgs/by-name/aw/awakened-poe-trade/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  appimageTools,
+  makeWrapper,
+  electron,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "awakened-poe-trade";
+  version = "3.25.102";
+
+  src = fetchurl {
+    url = "https://github.com/SnosMe/awakened-poe-trade/releases/download/v${version}/Awakened-PoE-Trade-${version}.AppImage";
+    hash = "sha256-lcdKJ+B8NQmyMsv+76+eeESSrfR/7Mq6svO5VKaoNUY=";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    name = "${pname}-${version}";
+    inherit src;
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin $out/share/${pname} $out/share/applications
+
+    cp -a ${appimageContents}/{locales,resources} $out/share/${pname}
+    cp -a ${appimageContents}/awakened-poe-trade.desktop $out/share/applications/
+    cp -a ${appimageContents}/usr/share/icons $out/share
+
+    substituteInPlace $out/share/applications/awakened-poe-trade.desktop \
+    --replace-fail 'Exec=AppRun' 'Exec=${pname}'
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+        makeWrapper ${lib.getExe electron} $out/bin/${pname} \
+    --add-flags $out/share/${pname}/resources/app.asar \
+    --prefix LD_LIBRARY_PATH : "${
+      lib.makeLibraryPath [
+        xorg.libXtst
+        xorg.libXt
+      ]
+    }"
+  '';
+
+  meta = {
+    description = "Path of Exile trading app for price checking";
+    homepage = "https://github.com/SnosMe/awakened-poe-trade";
+    changelog = "https://github.com/SnosMe/awakened-poe-trade/releases/tag/v${version}";
+    sourceProvenance = [ lib.sourceTypes.binaryBytecode ];
```

Comment:
What about it is bytecode?

Reply:
Side-note: should I ignore this warning?
```
❯ ./result/bin/awakened-poe-trade
hook_thread_proc [101]: Could not set thread priority 49 for thread 0x7F0BBEC7F6C0!
load_input_helper [1827]: XkbGetKeyboard failed to locate a valid keyboard!
APPIMAGE env is not defined, current application is not an AppImage
```